code/game/objects/items/devices/scanners/health.dm

```diff
@@ -304,15 +304,14 @@
 			var/blood_volume = H.vessel.get_reagent_amount("blood")
 			var/blood_percent =  round((blood_volume / H.species.blood_volume)*100)
 			var/blood_type = H.dna.b_type
-			var/blood_reagent = H.species.blood_reagents
```

Comment:
Looks like you are accidentally removing a feature here in this file

Reply:
Yep, just spotted that one, can readd that real quick